COSC481W-2024Fall/Ript-Fitness
# ciara/MyWorkoutsScreen

# My Workouts Screen 

## Description:

This PR implements the initial version of the My Workouts page. This page will store all of the completed workouts and written workouts that the user would like to save. After going to the Workouts tab, this screen will display when the "My workouts" button is pressed.

## Key Changes:

1. Created MyWorkouts screen
2. Created a "CustomSearchBar" for the search bar in the header
3. Created a search bar header with the title of the page and search branch
4. The main section of the screen is a safearea scrollview
5. Created a "PreCreatedWorkoutButton" to store the title and last completed date of the workout
6. Each button is hard coded at the moment
7. Each button will route to ApiScreen onPress for now - this will change in future issues

## What’s Not Completed:

1. The data for each button is hardcoded. Integration with the backend to pull real data will be completed in a future sprint
2. Situations with no workouts saved has not been dealt with 
3. The navigation is not set up 
4. The search bar is not functional 
5. The tests for my code 

## Testing Instructions:

1. The screen should scroll, minus the header part 
2. The buttons should each change opacity on hover 
3. On press each button should redirect 
4. Verify that the layout of the page adjusts appropriately on different screen sizes

Issue #10
